### PR TITLE
Upload 1364/smoke test nrss and eicr

### PR DESCRIPTION
**Pull request by cfarmer-fearless**

### CODE REVIEW

File: tests/smoke/kotlin/src/test/resources/invalid_manifests_invalid_value.json [hunk 1]
```diff
@@ -133,5 +133,34 @@
...
+    "version": "2.0",
+    "data_stream_id": "nrss",
+    "data_stream_route": "csv",
+    "sender_id": "blah",
+    "data_producer_id": "test-producer-id",
+    "jurisdiction": "test-jurisdiction",
+    "received_filename": "dex-smoke-test"
+  },
+  {
+    "version": "2.0",
```
Comment (AJKimpo) [inline tests/smoke/kotlin/src/test/resources/invalid_manifests_invalid_value.json:147]: shouldn't you include all the custom metadata fields for nrss-hl7v2? you included all the custom fields for eicr-fhir

File: tests/smoke/kotlin/src/test/resources/invalid_manifests_required_fields.json [hunk 2]
```diff
@@ -10,5 +10,23 @@
...
     "meta_ext_event": "testevent1"
+  },
+  {
+    "version": "2.0",
+    "data_stream_id": "nrss",
+    "data_stream_route": "csv",
+    "data_producer_id": "test-producer-id",
+    "jurisdiction": "test-jurisdiction",
+    "received_filename": "dex-smoke-test"
+  },
```
Comment (AJKimpo) [inline tests/smoke/kotlin/src/test/resources/invalid_manifests_required_fields.json:21]: This file is missing a section for nrss-hl7v2. Changes only include nrss-csv and eicr-fhir

**Review by AJKimpo — CHANGES_REQUESTED** [review]
left 2 comments on each of the invalid manifests files

File: tests/smoke/kotlin/src/test/resources/invalid_manifests_invalid_value.json (hunk 1 above)
Comment (cfarmer-fearless) [reply tests/smoke/kotlin/src/test/resources/invalid_manifests_invalid_value.json:147]: Fixed

**Review by AJKimpo — APPROVED** [review]
lgtm

**Pull request merged**